test/integration/fleetscope/fleetscope_test.go

```diff
@@ -41,11 +43,103 @@ func TestFleetscope(t *testing.T) {
+
+					if feature == "servicemesh" {
+						assert.Equal("MANAGEMENT_AUTOMATIC", gkeFeatureOp.Get("fleetDefaultMemberConfig.mesh.management").String(), fmt.Sprintf("Service Mesh Hub Feature %s should have mesh menagement equal to MANAGEMENT_AUTOMATIC", feature))
+					}
+
+					if feature == "multiclusteringress" {
+						for _, region := range clusterRegions {
+							membershipName := fmt.Sprintf("projects/%[1]s/locations/%[2]s/memberships/cluster-%[2]s-%[3]s", fleetProjectID, region, envName)
+							assert.Equal(membershipName, gkeFeatureOp.Get("spec.multiclusteringress.configMembership").String(), fmt.Sprintf("Service Mesh Hub Feature %s should have mesh menagement equal to MANAGEMENT_AUTOMATIC", feature))
+						}
+					}
+
+					// GKE Feature Membership
+					if feature == "configmanagement" {
+						for _, region := range clusterRegions {
+							fleetProjectNumber := gcloud.Runf(t, "projects describe %s", fleetProjectID).Get("projectNumber").String()
+							membershipName := fmt.Sprintf("projects/%[1]s/locations/%[2]s/memberships/cluster-%[2]s-%[3]s", fleetProjectNumber, region, envName)
+							configmanagementPath := fmt.Sprintf("membershipSpecs.%s.configmanagement", membershipName)
+
+							assert.Equal("gcpserviceaccount", gkeFeatureOp.Get(configmanagementPath+".configSync.git.secretType").String(), fmt.Sprintf("Hub Feature Membership %s should have git secret type equal to gcpserviceaccount", membershipName))
+							assert.Equal("unstructured", gkeFeatureOp.Get(configmanagementPath+".configSync.sourceFormat").String(), fmt.Sprintf("Hub Feature Membership %s should have source format equal to unstructured", membershipName))
+							assert.Equal("1.17.2", gkeFeatureOp.Get(configmanagementPath+".version").String(), fmt.Sprintf("Hub Feature Membership %s should have source format equal to unstructured", membershipName))
+							assert.Equal(rootReconcilerSa, gkeFeatureOp.Get(configmanagementPath+".configSync.git.gcpServiceAccountEmail").String(), fmt.Sprintf("Hub Feature Membership %s should have git service account type equal to %s", membershipName, rootReconcilerSa))
+							assert.True(gkeFeatureOp.Get(configmanagementPath+".policyController.enabled").Bool(), fmt.Sprintf("Hub Feature Membership %s  policy controler should be enabled", membershipName))
+							assert.True(gkeFeatureOp.Get(configmanagementPath+".policyController.referentialRulesEnabled").Bool(), fmt.Sprintf("Hub Feature Membership %s  referencial rule should be enabled", membershipName))
+							assert.True(gkeFeatureOp.Get(configmanagementPath+".policyController.templateLibraryInstalled").Bool(), fmt.Sprintf("Hub Feature Membership %s  template library should be installed", membershipName))
+						}
+					}
+				}
+
+				// GKE Membership binding
+				for _, region := range clusterRegions {
+					for _, clustersMembership := range multitenant.GetStringOutputList("cluster_membership_ids") {
```

Comment:
For projects with multiple clusterRegions, this nested `for` will result in multiple evaluations for each cluster.  Additionally, I believe it will fail for the invalid location `hub membership describe` calls.

I recommend retrieving the cluster's location from the `cluster_membership_id`, here is an example: https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/pull/52/files